hedera-node/hapi-utils/src/test/java/com/hedera/node/app/hapi/utils/ethereum/EthTxDataTest.java

```diff
@@ -44,6 +44,15 @@ class EthTxDataTest {
             "f864012f83018000947e3a9eaf9bcc39e2ffa38eb30bf7a93feacbc18180827653820277a0f9fbff985d374be4a55f296915002eec11ac96f1ce2df183adf992baa9390b2fa00c1e867cc960d9c74ec2e6a662b7908ec4c8cc9f3091e886bcefbeb2290fb792";
     static final String RAW_TX_TYPE_0_TRIMMED_LAST_BYTES =
             "f864012f83018000947e3a9eaf9bcc39e2ffa38eb30bf7a93feacbc18180827653820277a0f9fbff985d374be4a55f296915002eec11ac96f1ce2df183adf992baa9390b2fa00c1e867cc960d9c74ec2e6a662b7908ec4c8cc9f3091e886bcefbeb2290000";
+    // {
+    //  type: 1, to: '0x000000000000000000000000000000000000052D', data: '0x123456', nonce: 5644, gasLimit: '3000000',
+    //  gasPrice: '710000000000', maxPriorityFeePerGas: null, maxFeePerGas: null, value: '10000000000', chainId: '298',
+    //  r: '0xabb9e9c510716df2988cf626734ee50dcd9f41d30d638220712b5fe33fe4c816',
+    //  s: '0x249a72e1479b61e00d4f20308577bb63167d71b26138ee5229ca1cb3c49a2e53',
+    //  v: 28, accessList: []
+    // }
+    static final String RAW_TX_TYPE_1 =
+            "01f87182012a8085a54f4c3c00832dc6c094000000000000000000000000000000000000052d8502540be40083123456c001a07844f7acfe12e2985953cce823870b97464d7c2d5c752798c022b9403e0d5ca7a011154a57f1078fe7e36b6d17c8bb4a287cd385fc3168c2d43d24e193851f3186";
```

Comment:
perhaps it would be helpful to understand how you got to this string in the comments?  I see some of the values from your comment in this long hex number (but not all) like 01 and 000000000000000000000000000000000000052D but it's hard to know as a reviewer that the string is correct.

perhaps something like 
```
static final String RAW_TX_TYPE_1 =
"01" + // - type
"f87182012a8085a54f4c3c00832dc6c094" + // some random stuff having to do with rlp encoding?
"000000000000000000000000000000000000052" + // to
etc..?
```

Reply:
Here are the [RLP definitions](https://ethereum.org/en/developers/docs/data-structures-and-encoding/rlp/#definition). Should I add the torn-down tx in the PR?

Transaction hash:
```
01f87382012a82160c85a54f4c3c00832dc6c094000000000000000000000000000000000000052d8502540be40083123456c001a0abb9e9c510716df2988cf626734ee50dcd9f41d30d638220712b5fe33fe4c816a0249a72e1479b61e00d4f20308577bb63167d71b26138ee5229ca1cb3c49a2e53
```

And torn-down tx:
```
01 // type
f873 // total length
82 012a // chain id => 82 - 80 = 2 (hex) = 2 (dec) bytes length
82 160c // nonce  => same length
85 a54f4c3c00 // gas price => 5 bytes
83 2dc6c0 // gas limit => 3 bytes
94 000000000000000000000000000000000000052d // to => 94 - 80 = 14 (hex) = 20 (dec) bytes
85 02540be400 // value => 5 bytes
83 123456 // calldata => 3 bytes
c0 // empty access list => by the RLP definitions, an empty list is encoded with c0
01 // v
a0 abb9e9c510716df2988cf626734ee50dcd9f41d30d638220712b5fe33fe4c816 // r => a0 - 80 = 80 (hex) = 128 (dec) bytes
a0 249a72e1479b61e00d4f20308577bb63167d71b26138ee5229ca1cb3c49a2e53 // s => same
```

Reply:
Yes but let's do this in a follow up and maybe as part of the cherry pick since we got all of the signoffs and nothing functional or risky is left
@natanasow can you open a ticket with this to resolve in a follow up PR when you set the feature flag to true?